<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThirdWindow</class>
 <widget class="QMainWindow" name="ThirdWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>10</y>
      <width>611</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEdit_22"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Общий вид и тип</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_23"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Половой диморфизм</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="lineEdit_24"/>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>Голова</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="lineEdit_25"/>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>Глаза, выражение</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="lineEdit_26"/>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>Уши</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="lineEdit_27"/>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>Зубы</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>далее</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Детальное описание собаки</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
